cmd/lncli/commands.go

```diff
@@ -128,12 +128,12 @@ var newAddressCommand = cli.Command{
 	Action: actionDecorator(newAddress),
 }
 
-func newAddress(ctx *cli.Context) error {
-	ctxb := getContext()
-	client, cleanUp := getClient(ctx)
+func newAddress(cliCtx *cli.Context) error {
+	ctx := getContext()
+	client, cleanUp := getClient(cliCtx)
 	defer cleanUp()
 
-	stringAddrType := ctx.Args().First()
+	stringAddrType := cliCtx.Args().First()
```

Comment:
I'd prefer to keep the CLI context named as `ctx` and instead rename all `ctxb` to `ctxc` to make it more clear that it's backed by a cancel.